elastic/kibana
# [Reporting] Logging improvements while generating reports

## Summary

Closes https://github.com/elastic/kibana/issues/121588

* Added logic to extract the message from errors logged to the console (should help with future debugging of errors thrown to the console)
* Added logic to avoid warning about 3xx network responses returned to the browser

## Testing
1. Enable debug logging for `reporting` and `screenshotting` in kibana.yml
```
logging:
  loggers:
    - name: plugins.screenshotting
      level: debug
    - name: plugins.reporting
      level: debug
```
2. Run a PNG report of a dashboard, using a Vega panel
3. Any console messages or errors printed by the dashboard app to the JS console should be legible.
